resource.address/spi/src/main/java/org/kaazing/gateway/resource/address/Comparators.java

```diff
@@ -134,7 +135,7 @@ public int compare(ResourceAddress addr1, ResourceAddress addr2) {
             String authority1 = location1.getAuthority();
             String authority2 = location2.getAuthority();
             
-            int compareAuthority = STRING_COMPARATOR.compare(authority1, authority2);
+            int compareAuthority = CASE_INSENSITIVE_ORDER.compare(authority1, authority2);
```

Comment:
Do you want to wrap CASE_INSENSIVE_ORDER in compareComparable ? STRING_COMPARATOR is doing that and it further wraps in compareNonNull (so it protects from null authority values)